### ISSUE TITLE
How to run it?

### ISSUE THREAD

**Issue by leijue222** [Nohossat/simplon_bot#3]
> export FLASK_APP=chatbot_app
export FLASK_ENV=development
flask run

Win10 can't do it. Must it run in Linux?

**Comment by Nohossat** [comment]
Hi, sorry I didn't put the instructions for Windows only for MacOS. I think you have to replace **source venv/bin/activate** by **venv\Scripts\activate.bat** or **venv\Scripts\activate** to activate the environment then run the other instructions written in the README.md.

**Comment by leijue222** [comment]
Wow~ Your reply was very quick, thank you very much. Except for Windows, I can also try Ubuntu.
I already set up the requirements.txt environment by Conda.
Then I don't know how to run Flask. 
I don't see Flask code of  `app.run(host='127.0.0.1', port=5000, debug=False)`.

**Comment by Nohossat** [comment]
I was already in Github when you wrote so all good. Since it is a module, when you run this instruction **export FLASK_APP=chatbot_app**, Python will know the entry file is __init__.py and launch the app from there. The default host is 127.0.0.1 and default port is 5000. If you want to change the port, I think you can do it with **export FLASK_RUN_HOST=8080**

**Comment by leijue222** [comment]
Thanks, I will try it under Ubuntu.
And change the faq dataset to see the result.

**Comment by leijue222** [comment]
Nice application. But the accuracy does not seem to be very high. :confounded:
![QQ截图20210324222332](https://user-images.githubusercontent.com/30276789/112326605-9d5b2280-8cef-11eb-82c0-1fafa97b3566.png)

**Comment by Nohossat** [comment]
You are right...If I remember right it was around 60%. My comments for this project (it was a school project) were that maybe we could try other models and/or increase the number of comments in our training dataset. The dataset chosen also has the particularity of having a lot of irony (it is weird FAQ) so sometimes the results are quite unexpected haha.

**Comment by leijue222** [comment]
The FAQ is generally for a certain field, so there will not be too much data. It is estimated that it will be about a hundred.
I tried [RASA's method](https://rasa.com/docs/rasa/chitchat-faqs) in the past two days, but the result was a bit over-fitting. I changed the question sentence a little while keeping the meaning unchanged, but the answer was not what I asked.
I can't find a good method at present. Too sad..
